### ISSUE TITLE
Kepp Pokemon Above CP

### ISSUE THREAD

**Issue by Zerotask** [PocketMobsters/PokeMobBot#46]
I used Necrobot V.0.4.0 and set this value to 1500 after starting it, it transfered a 2440 Dragonite. Should this happen with this value set?
After that I looked up the config.json and saw in the array PokemonsNotToTransfer that there is no Dragonite, Lapras or Snorlax (best 3 non-legendarie)

**Comment by Esukho** [comment]
Me to , It Transfered my 1900 Dragonite :(

**Comment by StanleyDerLurch** [comment]
thats no issue... just entry the three names and ur dragonite stays in your inventory

**Comment by Zerotask** [comment]
I thought they are save, because they are above the value set in "KeepMinCp".

**Comment by StanleyDerLurch** [comment]
did u change this variable: "PrioritizeIvOverCp" to false?

**Comment by Zerotask** [comment]
The value is set to true.

**Comment by DeadElle** [comment]
```
    "Dragonite": {
      "KeepMinCp": 2600,
      "KeepMinIvPercentage": 90.0,
      "KeepMinDuplicatePokemon": 1
    }
```
edit this or add Dragonite in
`"PokemonsNotToTransfer": [`

**Comment by Anyrainel** [comment]
The list of overwrite default transfer config is so insane :(

**Comment by DeadElle** [comment]
@Anyrainel I'm the author of that list, it prioritizes IV over CP because of sniping, keeping a pokemon with high CP and low IV in your inventory is kinda unuseful because they're not worth to levelup/evolve at all

**Comment by Anyrainel** [comment]
@DeadElle I completely agree with prioritize IV over CP, but some of the CP threshold is not reasonable (very high as of the distribution), at least to all my lvl30-ish accounts. Also, people like me won't care to have more best-tier pokemons in bag, even if they'll never be powered up. At least you can use them before you have a 99 or 100 IV one.

**Comment by DeadElle** [comment]
@Anyrainel problem is that pkmn inventory is 250 slots only, CP is set too high with the purpose of not stacking high CP ones you find catching around. :) I suggest you setting PrioritizeIvOverCp to false and your problem will be solved ;)

**Comment by Anyrainel** [comment]
@DeadElle Thanks man. It is not me having problem. I customized those values. I don't want to lose high IV but low CP pms either. I just meant I would keep snorlax and dragonite above like 1800 CP rather than only over 2600 CP. There are not many snorlaxs and dragonites after all. Maybe it's I'm not having enough of them. Anyway I'm good and the system is good.
